netcdf/attributes.go

```diff
@@ -63,3 +78,16 @@ func (ds Dataset) Attr(name string) (a Attr) {
 func (ds Dataset) AttrN(n int) (a Attr, err error) {
 	return Var{ds, C.NC_GLOBAL}.AttrN(n)
 }
+
+// AddAttr adds a new a attribute named name and of value val.
+// The new attribute a is returned.
+func (v Var) AddAttrText(name string, val string) (err error) {
```

Comment:
Ok, changed it to the method AddAttr which takes one argument (the new attribute name) and return the attribute on which the WriteBytes method can be applied.